### ISSUE TITLE
Unable to install packages with `lib = path` 

### ISSUE THREAD

**Issue by JosiahParry** [r-wasm/webr#440]
I think the issue here is that webr tries to mount to an already existing location if the package has already been installed. If the path exists, it results in an error instead of installing into the path. 
Here we see that webr installs {cli} twice (instead of once) resulting in this error. 
```r
> webr::install("dplyr", lib = "/webrpkgs")
Downloading webR package: cli
Downloading webR package: generics
Downloading webR package: glue
Downloading webR package: cli
Error in mount(mountpoint, data_url) : 
  Unable to mount image, `/webrpkgs/cli` is already mounted.
```

**Comment by JosiahParry** [comment]
The error arises from this line of C code. 
https://github.com/r-wasm/webr/blob/c07d8997ddbb58f63002e8d46792be54edcd134c/packages/webr/src/mount.c#L32

**Comment by georgestagg** [comment]
Fixed by d8381511c88b6589ccfe72e2db25a0023a302c04, `webr::install()` was not taking into account the `lib` argument when searching for existing packages.

On /latest now:

```
> webr::install("dplyr", lib = "/webrpkgs")
Downloading webR package: cli
Downloading webR package: generics
Downloading webR package: glue
Downloading webR package: rlang
Downloading webR package: lifecycle
Downloading webR package: magrittr
Downloading webR package: fansi
Downloading webR package: utf8
Downloading webR package: vctrs
Downloading webR package: pillar
Downloading webR package: R6
Downloading webR package: pkgconfig
Downloading webR package: tibble
Downloading webR package: withr
Downloading webR package: tidyselect
Downloading webR package: dplyr
```

Thanks!

**Comment by JosiahParry** [comment]
Thank you so much for getting to these so quickly!!!
